save-common/src/commonMain/kotlin/org/cqfn/save/agent/TestExecutionDto.kt

```diff
@@ -0,0 +1,13 @@
+package org.cqfn.save.agent
+
+import kotlinx.serialization.Serializable
+import org.cqfn.save.domain.TestResultStatus
+
+@Serializable
+data class TestExecutionDto(
```

Comment:
We should introduce converted `TestExecution.fromDto` and `toDto` in `TestExecution` class and convert dto to entity in controller, dto shouldn't be passed to the service layer.